### PR TITLE
Use LESS and BEM implementation

### LINKED ISSUE #23: Use LESS and BEM
We had agreed, on the desktop client, to use LESS with the [BEM](https://getbem.com) methodology. We should also use both here.

### PR DESCRIPTION
**Pull request by Dawntraoz**
Create LESS structure. Add the global packages needed to run LESS compiler and autoprefixer to the Readme and push the generated styles in asset/css folder.
Add BEM methodology to name classes and structure LESS code.
PS: Exclude .idea in .gitignore (folder automatically generated by VS Code)
Implementation of ISSUE: #23 

fix #23

### CODE REVIEW

File: asset/css/main.css [hunk 1]
```diff
@@ -6,37 +6,30 @@
   --text: #000;
```
Comment (locness3) [inline asset/css/main.css:6]: Is this file the output of the LESS compiling?

File: README.md [hunk 2]
```diff
@@ -28,6 +28,12 @@ cd itchy-cordova
 npm install
 ```
 
+For styles we are using LESS, add it globally:
```
Comment (locness3) [inline README.md:31]: This is not needed, as we use a bundling tool, parcel, which compiles LESS automatically with it just being linked in the HTML like a CSS file. That also means you should remove the compiled CSS from the asset folder

**Review by locness3 — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (Dawntraoz) [reply README.md:31]: Oh okey, so can I also remove the lessc compiler from the build-styles tag??

What about autoprefixer plugin is being added by parcel too?

File: asset/css/main.css (hunk 1 above)
Comment (Dawntraoz) [reply asset/css/main.css:6]: Yes, but as I can see un the comment below, let's remove the folder and file 💃

File: README.md (hunk 2 above)
Comment (micahlt) [reply README.md]: I believe Parcel should handle autoprefixing as well, I'll look into it when I get a chance.  Thanks so much for your contribution!

**Pull request merged**